lms/static/scripts/frontend_apps/index.js

```diff
@@ -35,6 +35,16 @@ export function init() {
    */
   const services = new Map();
 
+  if (config?.debug?.values && Object.keys(config.debug.values).length) {
+    /* eslint-disable no-console */
+    console.groupCollapsed('Hypothesis debug');
```

Comment:
```suggestion
    console.groupCollapsed('Hypothesis debug info');
```

Sounds better as a heading with " info" on the end.